etc/taskcluster/docker/build.dockerfile

```diff
@@ -34,10 +34,6 @@ RUN \
         # blurz
         libdbus-1-dev \
         #
-        # Skia
-        libglu1-mesa-dev \
-        libbz2-dev \
```

Comment:
Are either of these still required for other reasons?

Reply:
Let’s find out!

Reply:
It doesn't appear that they are